src/functions/chat/Vulcan/index.js

```diff
@@ -0,0 +1,46 @@
+const io = require("socket.io-client");
+require("dotenv").config();
+
+const socket = io.connect(process.env.VULCANHUBURL);
+
+module.exports = async function(context, req) {
+  // All chat functions will receive a payload of:
+  // {
+  //    channel: string,
+  //    tags: {
+  //     badges?: Badges;
+  //     color?: string;
+  //     "display-name"?: string;
+  //     emotes?: { [emoteid: string]: string[] };
+  //     id?: string;
+  //     mod?: boolean;
+  //     turbo?: boolean;
+  //     'emotes-raw'?: string;
+  //     'badges-raw'?: string;
+  //     "room-id"?: string;
+  //     subscriber?: boolean;
+  //     'user-type'?: "" | "mod" | "global_mod" | "admin" | "staff";
+  //     "user-id"?: string;
+  //     "tmi-sent-ts"?: string;
+  //     flags?: string;
+  //     [paramater: string]: any;
+  // 'message-type'?: "chat" | "action" | "whisper";
+  // username?: string;
+  // bits?: string;
+  // },
+  //    message: string,
+  //    user: User
+  // }
+
+  const message =
+    "Vulcan is the name of the chatbot we use on stream, the source code for it can be found on our github repo at https://github.com/MichaelJolley/vulcan";
```

Comment:
Little proof-reading:

Vulcan is a server-less JavaScript Twitch chat bot that runs our stream.  It was built by our community and is completely open sourced at https://github.com/MichaelJolley/vulcan